apphub/image_classification/lenet_mnist.py

```diff
@@ -42,7 +43,7 @@ def get_estimator(epochs=2, batch_size=32):
     # step 3.prepare estimator
     traces = [
         Accuracy(true_key="y", pred_key="y_pred", output_name='acc'),
-        ModelCheckpoint('./lenet_model', monitor_name='acc', save_best_only=True)
+        LRController(model_name="lenet", lr_schedule=CyclicLRSchedule())
```

Comment:
if you're removing ModelCheckpoint, need to clean up the import statement

Reply:
not removing, i'm currently cleaning up the checkpoint code

Reply:
but you took it out of the trace list here. either it should be put back or the unused import should be removed

Reply:
fixed